pysteps/nowcasts/interface.py

```diff
@@ -90,6 +187,11 @@ def get_method(name):
     |  sseps          | short-space ensemble prediction system (SSEPS).       |
     |                 | Essentially, this is a localization of STEPS          |
     +-----------------+-------------------------------------------------------+
+    |  dgmr           | a deep generative model for the probabilistic    .    |
```

Comment:
why would you need to add this to the base pysteps package? Anything about DGMR should go within its plugin

Reply:
Goodmorning @dnerini , thanks for reviewing the pull request.

Well received, will remove that now.